### PR TITLE
data: sid-khuntwal

### PR DESCRIPTION
**Pull request by sid-khuntwal**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3164"></a>

### CODE REVIEW

File: sid-khuntwal.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/sidlyf/",
+      "icon": "FaLinkedin"
+    },
+    {
+    "name": "My Blogs",
+      "url": "https://sidlyf.com/",
+      "icon": "fa fa-firefox"
```
Comment (ChinmayMhatre) [inline sid-khuntwal.json:26]: Hey @sid-khuntwal , I think the formatting of the icon name is wrong.

 you can go here and look for the icon you want https://linkfree.eddiehub.io/icons

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please move your profile to the data directory

**Review by dan-mba — CHANGES_REQUESTED** [review]
There are now no files in this PR. It looks like you deleted your profile instead of moving it.

**Comment by sid-khuntwal** [comment]
opps, sorry, let me just create a new PR .

**Pull request closed without merging**